pkg/overlay/overlay_test.go

```diff
@@ -11,28 +11,45 @@ import (
 
 const (
 	schemaFile               = "testdata/base.yaml"
-	schemaJSON               = "testdata/base.json"
-	overlayFile              = "testdata/overlay.yaml"
+	schemaJSON    = "testdata/base.json"
+	overlayFileV1 = "testdata/overlay.yaml"
+	overlayFileV2 = "testdata/overlay-v2.yaml"
```

Comment:
Super nit: Might just be me, but wonder if for future travelers we should name these with "rfc9535" or "jsonpath-rfc9535" instead of "v2" (and consider "v1" -> "jsonpath-vmware") because whilst v2 makes sense at this moment it has nothing to do with a v2.0.0 of the overlay spec.